app/models/view/reports/Report.java

```diff
@@ -166,5 +179,6 @@ public String toString() {
     private String _name;
     private ReportSource _source;
     private Schedule _schedule;
+    private Duration _timeout;
```

Comment:
I'm guessing for the creation form the default timeout should be set here (assuming the form is rendered from a default instance of the view model).

Reply:
AFAICT it's not: when I visit the Reports page or when I click "Create new report", I see no network request that looks like it grabs the default view model. So I think it just instantiates the EditReportViewModel.

I changed [the EditReportViewModel's default for `timeoutString`](https://github.com/ArpNetworking/metrics-portal/pull/250/files#diff-98ce7240ffe1e8361a3005e708e6bbe8R44), and that had the desired effect in the UI.